packages/examples_support/src/load_generator/generator.rs

```diff
@@ -93,6 +94,8 @@ impl ControlledRateLoadGenerator {
             }
         }
 
+        tx_output.close();
```

Comment:
👍

Reply:
Still a bit confused as to why `drop` was not working while exiting the generator. But if explicit `close` is working that should be good. 

![image](https://github.com/kindredgroup/talos/assets/118979108/56729e45-459f-4a25-a76d-67874ed38afb)

Reply:
Because its arc'ed?

Reply:
I did check that during the review, Arc should drop the inner value if the reference count goes to 0. 

And in our use case, we don't clone it. So the reference count is 1 which on exiting will become 0, and therefore drop the inner value.

![image](https://github.com/kindredgroup/talos/assets/118979108/9688ce21-33e0-47d1-8fce-7e847ef20630)